### PR TITLE
No Bug - Disable capabilities for running on device

### PR DESCRIPTION
**Pull request by mauryat**
Disable Push Notifications capability for running on device with Xcode developer account

### CODE REVIEW

**Comment by mauryat** [comment]
@sleroux, could you merge this if it looks alright?

File: Docs/BUILDING.md [hunk 1]
```diff
@@ -4,15 +4,15 @@ Building Firefox for iOS
...
+* Xcode 8.2.1 with the iOS 10 SDK (Betas not supported)
 * Carthage 0.15 or newer
 
 When running on a device:
 
 * A device that supports iOS 9.3 GM or later
 * One of the following:
  * A developer account and Admin access to the *Certificates, Identifiers & Profiles* section of the *iOS DevCenter*
- * A free developer account, new with Xcode 8
+ * A free developer account, new with Xcode 7
```
Comment (sleroux) [inline Docs/BUILDING.md]: Should this be Xcode 8 as well?

**Comment by sleroux** [comment]
Thanks for adding this! I completely forgot about the issues with push notifications and personal certs. Just need to revert the references to Xcode 7 and I'll go ahead and merge.

**Comment by mauryat** [comment]
@sleroux, I removed references to Xcode 7. I guess that was creating unnecessary confusion. Apple enabled free developer accounts from Xcode 7 onwards. But, since we use Xcode 8, we don't need to talk about Xcode 7 anymore.

**Pull request merged**